app/config/settings_dev.py

```diff
@@ -22,6 +22,10 @@
     strtobool(os.getenv('DEBUG_ENABLE_DB_EXPLAIN_ANALYZE', 'False'))
 )
 
+RUN_LARGE_ASSET_FILE_UNIT_TEST = bool(
+    strtobool(os.getenv('RUN_LARGE_ASSET_FILE_UNIT_TEST', 'True'))
+)
+
```

Comment:
I would not differentiate unit test between dev and prod, this don't make sense, unit test are for local dev, and are to avoid merging, deploying non working code. This settings should go then in settings_prod.py for simplicity.

Reply:
Now this whole problem is fixed in a much shorter way